src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,113 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int defaultCapacity = 16;
+    private static final double loadFactor = 0.75;
+    private Node<K, V>[] buckets;
+    private int size;
+
+    public MyHashMap() {
+        buckets = new Node[defaultCapacity];
+    }
 
     @Override
     public void put(K key, V value) {
+        putValue(hash(key), key, value);
+    }
 
+    private void putValue(int hash, K key, V value) {
+        if (size == buckets.length * loadFactor) {
+            resize();
+        }
+        Node<K, V> putNode = new Node(key, value, null);
```

Comment:
```suggestion
        Node<K, V> newNode = new Node<>(key, value, null);
```
better naming, and proper usage of diamond operator, read more about it on common mistakes page